### ISSUE TITLE
App throws when bundling with --uglify and using FadeTransition (for RadSideDrawer)

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/nativescript-dev-webpack#258]
**Steps to reproduce** - use [this application](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/others/t1125353) or follow these steps:

- create new app with 
```
tns create myApp --template https://github.com/NativeScript/template-drawer-navigation-ng
```

- install WebPack
```
npm install --save-dev nativescript-dev-webpack
npm install
```

- bundle with uglify
```
npm run start-android-bundle --uglify
```

**Result**: the application is build succsessfully but then throws the following:
```
JS: Warning: 't' is not supported when 'showOverNavigation' is set to 'true'.
JS: ERROR RangeError: Maximum call stack size exceeded
```

Reproducible with both official and next releases of tns-core-modules (official  3.1.1) and nativescript-dev-webpack (official 0.7.3).

reported via t1125353

**Comment by elvismercado** [comment]
i also have this issue with `npm run start-ios-bundle --uglify`

**Comment by ChAraya** [comment]
same here but also have problem with the CSS in the sidebar.

here some examples:

with tns run ios

![captura de pantalla 2017-09-07 a la s 10 22 33](https://user-images.githubusercontent.com/19436477/30165460-8669b5d6-93b6-11e7-967d-03c178bbe3a0.png)

with npm run start-ios-bundle --uglify and npm run start-ios-bundle

![captura de pantalla 2017-09-07 a la s 10 20 15](https://user-images.githubusercontent.com/19436477/30165489-9f6d034e-93b6-11e7-9267-076f45a6906e.png)

**Comment by pako8x** [comment]
try 
`npm install -g npm-check-updates`
`ncu -u`
`npm install`

and then
`npm run start-android-bundle --uglify`

It worked for me

**Comment by mhadaily** [comment]
I have the same issue even with other transition such `public sideDrawerTransition: DrawerTransitionBase = new PushTransition();`

tns-ios version: 3.4
nativescript-pro-ui version: 3.3.0 

thanks, guys.

**Comment by felipebueno** [comment]
Hi, @ChAraya! 

Did you manage to solve the CSS problem with the sidedrawer? I'm facing the same issue at this moment and I have no clue on how to fix that.

With webpack 👎 
![drawer_css_with_webpack](https://user-images.githubusercontent.com/241271/35452516-e2e86468-02ad-11e8-8a8c-74021c220fa2.png)


Without webpack 👍 
![drawer_css_without-webpack](https://user-images.githubusercontent.com/241271/35452495-cc1f2cee-02ad-11e8-8165-386583b7426d.png)

**Comment by sis0k0** [comment]
@felipebueno, can you share your project or some steps to reproduce the issue? The css looks if I use the [drawer template](https://market.nativescript.org/plugins/tns-template-drawer-navigation-ng) and build the app with `tns run android --bundle --env.uglify`.

**Comment by felipebueno** [comment]
Ok, just a follow up on my [previous comment](https://github.com/NativeScript/nativescript-dev-webpack/issues/258#issuecomment-360850479).

As a temporary solution, I moved all the custom styles from the ```app/shared/my-drawer/_MyDrawer.scss``` file to the ```app/_app-common.scss``` and it works as expected.

@sis0k0 Hmm, I can't share the project (becaus it's not mine) but I'll create a new project using the ```template-drawer-navigation``` template and try to reproduce the problem. I'll comment here again as soon as I publish it to a repository so you can take a look.

Thanks, @sis0k0 😃

**Comment by felipebueno** [comment]
@sis0k0 I managed to get it working :)

I'm not sure what was the problem but here's what I did.

I created a new project with the drawer-navigation template, bundled it with webpack and run it on my device. Everything worked properly, including the drawer's styles.

Then I copied the drawer module from this new project to my main project (that one from the screenshots above) and updated all my pages to use this new ```app/shared/drawer``` module instead of the old ```app/shared/my-drawer```. I bundled the app and it woked :)

I have no idea why this worked but I have some guesses. I noticed that the new project I created with the drawer template was different from the one I created three weeks ago which is the one I'm using to build my project. Maybe they fixed some bugs, I don't know... will look into it another time. Getting back to work now 😄
